documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+  fill in the details in this area?"
+
+In short, the goal of the question round is for everyone participating to fully
+understand the proposal. Participants are urged that many objections can be
+restated as questions. This helps promote good faith between participants and
+allows the author to clarify things they might have missed in the initial draft
+of the proposal. This does _not_ mean that known paramount objections should be
+reserved by participants. If you know that a proposed solution is impossible or
+illegal, please do raise this concern as soon as possible.
+
+Note that the author does not have to respond to every question. However, they
+should keep them in mind during the revision round.
+
+#### Reactions
+
+This round also invites participants to share reactions, positive or negative,
+separate from potential paramount objections and in particular is an excellent
+venue to highlight positive aspects of the proposal regardless of whether
+paramount objections might be raised later. You may also share non-paramount
+objections at this round. For example, “I find JavaScript harder to work with
+than Python” is an objection but not a paramount objection. What makes an
+objection “paramount” is clarified further in the
+[Objection round](#objection-round) below.
+
+Reactions are explicitly expected at this round because, similar to questions,
+they can help prompt further clarification from the author to be address in the
+revision round.
+
+#### Purpose
+
+Proposals should all include an explicit "purpose". Often these are informed by
+the needs of a particular project associated with the proposal or with a
+requested discussion as the result of a retrospective or other self-feedback
+mechanism. However, contributors may, at their discretion, raise proposals for
+discussion that do not have prior conversations. In these cases, the question
+round is also understood to include an evaluation and clarification of the
+purpose of the proposal. For regularly requested proposals, however, the purpose
+is taken to be accepted and should not be the focus
+
+### Revision round
+
+At this round, the author has the opportunity to revise the proposal in response
+to the questions and reactions raised. This round has no set time period. The
+author should use their discretion and give an expected time when the revised
+version will be ready. The expected date of completion for the revision is both
+for accountability (to prevent projects from being stalled indefinitely) and for
+clarity of expectations for the other participants in the process. If the author
+requires additional time beyond their initial estimation, they should let the
+other participants know as soon as possible, keeping in mind that other
+discussions may be stalled in the meantime.
+
+To every extent possible, **no revisions should be made to the proposal before
+this point other than grammatical or spelling corrections**. The reason for this
```

Comment:
```suggestion
**no revisions should be made to the proposal before
this point other than grammatical or spelling corrections**. The reason for this
```

We can avoid clarifications like "To every extent possible," since you explained the imperative language used at the start of the document.